SampleApp-Kotlin/src/main/java/com/paypal/example/paypalandroidsdkexample/SampleActivity.kt

```diff
@@ -0,0 +1,304 @@
+package com.paypal.example.paypalandroidsdkexample
+
+import com.paypal.android.sdk.payments.PayPalAuthorization
+import com.paypal.android.sdk.payments.PayPalConfiguration
+import com.paypal.android.sdk.payments.PayPalFuturePaymentActivity
+import com.paypal.android.sdk.payments.PayPalItem
+import com.paypal.android.sdk.payments.PayPalOAuthScopes
+import com.paypal.android.sdk.payments.PayPalPayment
+import com.paypal.android.sdk.payments.PayPalPaymentDetails
+import com.paypal.android.sdk.payments.PayPalProfileSharingActivity
+import com.paypal.android.sdk.payments.PayPalService
+import com.paypal.android.sdk.payments.PaymentActivity
+import com.paypal.android.sdk.payments.PaymentConfirmation
+import com.paypal.android.sdk.payments.ShippingAddress
+
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.TextView
+import android.widget.Toast
+
+import org.json.JSONException
+
+import java.math.BigDecimal
+import java.util.Arrays
+import java.util.HashSet
+
+/**
+
+ * THIS FILE IS OVERWRITTEN BY `androidSDK/src/|partner>sampleAppJava.
```

Comment:
i think it was there since ever to let us know that this file was getting copied over everytime we release. Removing it.